test/python/test_user_config.py

```diff
@@ -131,6 +129,7 @@ def test_all_options_valid(self):
         [default]
         circuit_drawer = latex
         circuit_mpl_style = default
+        circuit_mpl_style_path = ~;~/.qiskit
```

Comment:
Does `~` means `$HOME`? Does this works in Windows?

Reply:
First off, it passes all the auto tests. I'm using this (from the os.path docs)
`os.path.expanduser(path)`
`On Unix and Windows, return the argument with an initial component of ~ or ~user replaced by that user’s home directory.`
It does leave the question of the Mac, but it does pass the Mac tests. So not sure if it's universal.
It's just a testing issue. A user would put in the appropriate directory format for their system. Is there a more generic way to put in 2 dir names?

Reply:
I think it's safe. We used the same approach in other parts. 

> Is there a more generic way to put in 2 dir names?

I would use `:`. It's the `$PATH` way.

Reply:
Ah, good catch. I'll change that.

Reply:
Done in https://github.com/Qiskit/qiskit-terra/pull/5223/commits/bfb8be30672ba6d81187f57c58a4079026e0bb0e